app/syncables_test.go

```diff
@@ -70,32 +72,17 @@ func TestPopulateSyncablesSince(t *testing.T) {
 		t.Errorf("test group not created: %s", err.Error())
 	}
 
-	_, err = th.App.CreateGroupSyncable(&model.GroupSyncable{
-		AutoAdd:    true,
-		GroupId:    gleeGroup.Id,
-		SyncableId: practiceChannel.Id,
-		Type:       model.GroupSyncableTypeChannel,
-	})
+	_, err = th.App.CreateGroupSyncable(model.NewGroupChannel(gleeGroup.Id, practiceChannel.Id, true))
```

Comment:
The changes like this in this file are just switching to use the `NewGroupChannel` and `NewGroupTeam` helper functions.